roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -28,11 +28,10 @@ cloud:
 discovery:
   type: kubernetes
   zen.ping.multicast.enabled: false
-  zen.minimum_master_nodes: {{es_min_masters}}
+  zen.minimum_master_nodes: {{es_node_quorum}}
```

Comment:
So as an operator, I want to be able to update my deployment configuration for my ES pod and redeploy the pod.  If I add nodes, I want to be able to add the new node, and then gracefully update the other nodes to have the same configuration.  As long as I can redeploy one pod at a time to pick up the new configuration, we're good.  So if that means keepings these as environment variables as provided by the deployment configuration, great.